hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1687,9 +1689,15 @@ class VCFsReader(
     TStruct.empty(),
     colType = TStruct("s" -> TString()),
     colKey = Array("s"),
-    rowType = kType ++ header1.vaSignature,
+    rowType = kType ++ header1.vaSignature.virtualType,
     rowKey = Array("locus"),
-    entryType = header1.genotypeSignature)
+    entryType = header1.genotypeSignature.virtualType)
+
+  val fullRVDType = RVDType(PCanonicalStruct(false,
```

Comment:
could be useful to factor this out into a function called here and above, not blocking, just a bit less error prone since this assignment is complicated.

Reply:
good change, will do that later. Don't want to interrupt the agg stuff right now.